docs_sources/erp-eml-epa-notes.adoc

```diff
@@ -0,0 +1,130 @@
+
+Jede Stelle aus den Profilen KBV_PR_ERP_Medication_FreeText, KBV_PR_ERP_Medication_Ingredient und KBV_PR_ERP_Medication_PZN kann in das neue Profil GEM_ERP_PR_Medication gemappt werden. Andernfalls wird ein Hinweis angegeben, dass der entsprechende Wert nicht übernommen wird.
+
+=== Erzeugen von Medications für Rezepturen und Kombipackungen
+
+Die Darstellung von Rezepturen und Kombipackungen in der ePA unterscheidet sich zum Profil KBV_PR_ERP_Medication_Compounding.
+
+In KBV_PR_ERP_Medication_Compounding sind alle Bestandteile einer Rezeptur in _einer_ Medication als PZN Codes unter .ingredient aufgelistet. Dies bietet Vorteile für die Datensparsamkeit und reduziert die Komplexität der Daten.
+
+Für die Abgabe von Rezepturen hat das technisch u.A. den Nachteil, dass nicht für jeden PZN-Bestandteil der Rezeptur eine Angabe zur Chargeninformation angegeben werden kann.
+
+Die ePA bietet folgende Profile mit Besonderheiten:
+
+[cols="1,2"]
+|===
+|Profil|Besonderheit
+
+|link:https://simplifier.net/epa-medication/epamedication[EPA Medication] a|
+* Generisches Medication Profil
+* GEM_ERP_PR_Medication ist hiervon abgeleitet
+* Kann Kindelemente als .contained enthalten
+|link:https://simplifier.net/epa-medication/epamedicationpzningredient[EPA Medication Ingredient] a|
+* Medication zur Angabe eines Bestandteils einer Rezeptur
+* Kann keine weiteren Medications unter .ingredient.itemReference enthalten
+|link:https://simplifier.net/epa-medication/epamedicationpharmaceuticalproduct[EPA Pharmaceutical Product Medication] a|
+* Medication zur Angabe eines pharmazeutischen Produkts
+* Enthält den Produktschlüssel unter .code.coding:product-key
+
+|===
+
+==== Rezepturen
+
+Für Rezepturen wird genau eine EPA Medication erzeugt. Diese Medication enthält unter .extension:type den Wert "Extemporaneous preparation (product)". Damit ist diese Medication als Rezeptur gekennzeichnet.
+
+Für jeden Bestandteil der Rezeptur wird  eine EPA Medication Ingredient erzeugt und als .contained hinzugefügt. Die EPA Medication Ingredient Objekte können keine weiteren Medications enthalten, sondern nur die Angaben zu einem Bestandteil einer Rezeptur.
+
+Schematisch stellt sich eine Rezeptur wie folgt dar:
+
+image:medication-rezeptur-schematics.png[width=50%]
+
+.Beispiel eines Rezeptur FHIR-Datensatzes (Klicken zum Ausklappen)
+[%collapsible]
+//TODO: Change to real example from Examples Repository
+====
+[source,xml]
+----
+include::../resources/temp/Medication-Medication-Rezeptur.xml[]
+----
+====
+
+==== Kombipackung
+
+Das Erzeugen von Kombipackungen geschieht analog zur Rezeptur. Statt der Ingredient Profile wird das Profil EPA Medication Pharmaceutical Product für die Angabe der Bestandteile einer Kombipackung genutzt.
```

Comment:
Macht es Sinn hier auf das Tooling des [BfArM Data Explorer ](https://bfarm-referenzdatenbank-explorer-frontend.onrender.com/) zu verlinken um darauf zu verweisen, wie Kombiepackungen generisch ausssehen sollen?
Aktuell fehlt noch ein finales Review, ob die Ressourcen korrekt sind. 

Was fehlt noch? Eine spezifische Benennung de GEM_ERP_PR_Medication in Dropdown?